rlog/2024-07-01-bloomfilter.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Membership with Bloom Filters'
+date: 2024-06-13 12:00:00
+authors: marvin
+published: false
+slug: dr
+categories: research
+
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Improving on the previous version of RLN by allowing dynamic epoch sizes.
```

Comment:
Thanks for your suggestion for adding Cuckoo filters. This line about RLN was accidentally left in from another rlog (yours) that I copied to get started. But, I think having Bloom filters, Cuckoo filters and tying Cuckoo filters make for a much better rlog!

Reply:
unclear here if the filter should be used to store memberships or messages, we can discuss this out of band since this is not in the scope of the rlog 😉